src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -675,7 +675,7 @@ case object CREATE extends OpCode(0xf0, 3, 1, _.G_create) {
     } else {
 
       val (initCode, memory1) = state.memory.load(inOffset, inSize)
-      val (newAddress, world1) = state.world.newAddress(state.env.ownerAddr)
+      val (newAddress, world1) = state.world.increaseCreatorNonceAndCreateAddress(state.env.ownerAddr)
```

Comment:
I'm not completely sure about this in terms of having a single function that does this. I would create an `increaseNonce` fn and then creating the address. That would look more similar to YP `CREATE` rule that states `σ ∗ ≡ σ except σ ∗ [I a ] n = σ[I a ] n + 1`.

But, I would like to stick to @rtkaczyk preference on this. @rtkaczyk are you ok with this?